polaris-react/src/components/IndexTable/IndexTable.tsx

```diff
@@ -151,59 +150,52 @@ function IndexTableBase({
+        }));
 
-          if (tableHeadings.current.length === 0) {
-            return;
-          }
+        if (tableHeadings.current.length === 0) {
+          return;
+        }
+
+        // update left offset for first column
+        if (selectable && tableHeadings.current.length > 1)
+          tableHeadings.current[1].style.left = `${tableHeadingRects.current[0].offsetWidth}px`;
 
-          // update left offset for first column
-          if (selectable && tableHeadings.current.length > 1)
-            tableHeadings.current[1].style.left = `${tableHeadingRects.current[0].offsetWidth}px`;
+        // update the min width of the checkbox to be the be the un-padded width of the first heading
+        if (selectable && firstStickyHeaderElement?.current) {
+          const elementStyle = getComputedStyle(tableHeadings.current[0]);
+          const boxWidth = tableHeadings.current[0].offsetWidth;
+          firstStickyHeaderElement.current.style.minWidth = `calc(${boxWidth}px - ${elementStyle.paddingLeft} - ${elementStyle.paddingRight} + 2px)`;
+        }
 
-          // update the min width of the checkbox to be the be the un-padded width of the first heading
-          if (selectable && firstStickyHeaderElement?.current) {
-            const elementStyle = getComputedStyle(tableHeadings.current[0]);
-            const boxWidth = tableHeadings.current[0].offsetWidth;
-            firstStickyHeaderElement.current.style.minWidth = `calc(${boxWidth}px - ${elementStyle.paddingLeft} - ${elementStyle.paddingRight} + 2px)`;
+        // update sticky header min-widths
+        stickyTableHeadings.current.forEach((heading, index) => {
+          let minWidth = 0;
+          if (index === 0 && (!isSmallScreen() || !selectable)) {
+            minWidth = calculateFirstHeaderOffset();
+          } else if (selectable && tableHeadingRects.current.length > index) {
+            minWidth = tableHeadingRects.current[index]?.offsetWidth || 0;
+          } else if (!selectable && tableHeadingRects.current.length >= index) {
+            minWidth = tableHeadingRects.current[index - 1]?.offsetWidth || 0;
           }
 
-          // update sticky header min-widths
-          stickyTableHeadings.current.forEach((heading, index) => {
-            let minWidth = 0;
-            if (index === 0 && (!isSmallScreen() || !selectable)) {
-              minWidth = calculateFirstHeaderOffset();
-            } else if (selectable && tableHeadingRects.current.length > index) {
-              minWidth = tableHeadingRects.current[index]?.offsetWidth || 0;
-            } else if (
-              !selectable &&
-              tableHeadingRects.current.length >= index
-            ) {
-              minWidth = tableHeadingRects.current[index - 1]?.offsetWidth || 0;
-            }
-
-            heading.style.minWidth = `${minWidth}px`;
-          });
-        },
-        SIXTY_FPS,
-        {leading: true, trailing: true, maxWait: SIXTY_FPS},
```

Comment:
When these values are omitted debounce will fallback to RAF.

Sixty fps is aprox 16.7ms which is how often RAF will fire. Rather than estimate when the frame will be, let's use it!